compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/InterfaceLowering.kt

```diff
@@ -67,6 +80,33 @@ private class InterfaceLowering(val context: JvmBackendContext) : IrElementTrans
         Visibilities.isPrivate(function.visibility) && function.name != clinitName ||
                 function.origin == IrDeclarationOrigin.FUNCTION_FOR_DEFAULT_PARAMETER ||
                 function.origin == JvmLoweredDeclarationOrigin.SYNTHETIC_METHOD_FOR_PROPERTY_ANNOTATIONS
+
+    private fun IrFunction.isJvmDefault(): Boolean =
+        (hasAnnotation(JVM_DEFAULT_FQ_NAME) ||
+                (this is IrSimpleFunction && correspondingPropertySymbol?.owner?.hasAnnotation(JVM_DEFAULT_FQ_NAME) == true)) &&
+                context.state.target.bytecodeVersion >= JvmTarget.JVM_1_8.bytecodeVersion &&
+                context.state.jvmDefaultMode.isEnabled &&
+                origin != JvmLoweredDeclarationOrigin.SYNTHETIC_METHOD_FOR_PROPERTY_ANNOTATIONS
+
+    private fun callDefaultImpls(defaultImpls: IrFunction, interfaceMethod: IrFunction): IrCall {
+        val startOffset = interfaceMethod.startOffset
+        val endOffset = interfaceMethod.endOffset
+
+        return IrCallImpl(interfaceMethod.startOffset, interfaceMethod.endOffset, interfaceMethod.returnType, defaultImpls.symbol).apply {
+            passTypeArgumentsFrom(interfaceMethod)
+
+            var offset = 0
+            interfaceMethod.dispatchReceiverParameter?.let {
```

Comment:
```        
       val receiverAndArgs = interfaceMethod.receiverAndArgs()
        receiverAndArgs.forEachIndexed { i, irExpression ->
        newExpression.putValueArgument(i, irExpression)
```
        }

Reply:
`.receiverAndArgs()` seems only available for `IrFunctionAccessExpression` (call site) but not `IrFunction` (definition). Looks like `IrGetValueImpl()` has to be called explicitly?

Reply:
Yes, sorry for mistake